### PR TITLE
New Webscraper Automation Script

### PR DESCRIPTION
**Pull request by ayman-m**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
An Automation script to scrap web pages

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x ] No

## Must have
- [ ] Tests
- [x ] Documentation

### CODE REVIEW

**Comment by ayman-m** [comment]
I am going to submit another PR of the new container image referenced here

**Comment by daryakoval** [comment]
@ayman-m  Hi, We have docker image [bs4](https://github.com/demisto/dockerfiles/tree/master/docker/bs4), that uses lxml and beautifulSoup libraries. Maybe it is possible to use it instead?

File: Packs/WebScraper/Scripts/WebScraper/WebScraper.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+from typing import Dict, Any
+import traceback
+from bs4 import BeautifulSoup
+from operator import attrgetter
+import xmltodict
```
[the remark this answers is not in the document]
Comment (daryakoval) [reply Packs/WebScraper/Scripts/WebScraper/WebScraper.py:10]: Ok, you are right. Please see my comments at [docker pr](https://github.com/demisto/dockerfiles/pull/6323#pullrequestreview-773576422).

File: Packs/WebScraper/Scripts/WebScraper/WebScraper.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        results = client.webscrap_url(headers=headers, params=params, navigator_tree=navigator_tree)
+    else:
+        raise ValueError('Please use page_url or page_html arguments to start scraping')
+
+    content_outputs = {
+        "Tree": results
+    }
+    return CommandResults(
+        outputs_prefix='WebScraper',
+        outputs_key_field='',
```
Comment (daryakoval) [inline Packs/WebScraper/Scripts/WebScraper/WebScraper.py:73]: Every time that someone will execute this script, all context data related to `WebScraper` will be replaced. Is it ok?

File: Packs/WebScraper/Tests/id_set.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (daryakoval) [inline Packs/WebScraper/Tests/id_set.json:1]: What is this file? 
If was uploaded by mistake please remove.

**Review by daryakoval — CHANGES_REQUESTED** [review]

File: Packs/WebScraper/Scripts/WebScraper/WebScraper.yml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  description: Scraped Pages
+  type: String
+script: '-'
+subtype: python3
+system: false
+tags:
+- ''
+timeout: '0'
+type: python
+dockerimage: aymanam/scraper:latest
```
[the remark this answers is not in the document]
Comment (ayman-m) [reply Packs/WebScraper/Scripts/WebScraper/WebScraper.yml:60]: @daryakoval I tested the new image, it works fine

[the remark this answers is not in the document]
Comment (daryakoval) [reply Packs/WebScraper/Scripts/WebScraper/WebScraper.yml:60]: Please use new docer image: `demisto/bs4-py3:1.0.0.24176`

**Comment by daryakoval** [comment]
@ayman-m Let's schedule a demo session.

**Review by daryakoval — APPROVED** [review]

**Pull request merged**